src/modules/kvs/kvstxn.c

```diff
@@ -68,15 +68,15 @@ struct kvstxn {
      * INIT -> LOAD_ROOT
      * LOAD_ROOT -> APPLY_OPS
      * APPLY_OPS -> STORE
-     * STORE -> PRE_FINISHED
-     * PRE_FINISHED -> FINISHED
+     * STORE -> POST_STORE
```

Comment:
Is there a better name that would describe what's actually happening in the state rather than just "the state that comes after store"?

Reply:
good idea, lets go with "generate keys"